Scripts/Utilities/MaterialUtils.cs

```diff
@@ -13,7 +13,7 @@ namespace UnityEditor.Experimental.EditorVR.Utilities
 	static class MaterialUtils
 	{
 		/// <summary>
-		/// Get a material clone; IMPORTANT: Make sure to call U.Destroy() on this material when done!
+		/// Get a material clone; IMPORTANT: Make sure to call ObjectUtils.Destroy() on this material when done!
```

Comment:
Hey, while you're at it, want to finish documenting this class? ;)

Reply:
Not in this PR ;-)